src/data_write.py

```diff
@@ -1267,7 +1267,7 @@ def main():
                                       file_ext=args.file_type,
                                       aveperiod_meta=aveperiod_metadata,
                                       data_parsing=data_parsing,
-                                      rt_dw={"socket": realtime_to_data_write,
+                                      rt_dw={"socket": data_write_to_realtime,
                                              "iden": options.rt_to_dw_identity},
```

Comment:
If it really is data_write -> realtime, we should rename the corresponding variables. I.e. rt_dw -> dw_rt, and the options identity names

Reply:
It is only one-way, from data_write to realtime. I will review and update names where suitable.

Reply:
Ah. Slight misunderstanding of how this worked on my part. Each identity is really just a link between the two processes, so calling it `module1_to_module2_identity` is a bit confusing. So if `module1` is `dw` and `module2` is `rt`, then both identities are needed. `dw_to_rt_identity` identifies the socket that data_write creates for communicating with realtime, and `rt_to_dw_identity` identifies the socket that realtime creates for communicating with data_write. It doesn't matter that realtime only receives; it still needs that socket. Each socket actually connects the process with the router, hence there being two sockets for each communication.

Reply:
Tying this into your original comment: `options.rt_to_dw_identity` is thus specifying that the information sent by data_write is to be forwarded through the socket that realtime creates that is for connection to data_write.